lib/season_stats.rb

```diff
@@ -9,49 +9,52 @@ def initialize(file_path)
     @game_teams_collection = file_path[:game_teams_collection]
   end
 
-  # def games_by_coach(head_coach)
-  #   @game_teams_collection.game_teams.find_all do |game_team|
-  #     game_team.head_coach == head_coach
-  #   end
-  # end
   def games_by_season(season)
     @games_collection.games.find_all do |game|
       game.season == season
     end
   end
 
+  def game_ids_by_season(season)
+    games_by_season(season).map do |game|
+      game.game_id
+    end
+  end
+
   def game_teams_by_season(season)
-    acc = []
```

Comment:
We can double check if these helper methods are needed anywhere, otherwise this looks good.